### PR TITLE
Translate legacy conv ref to next-gen ones

### PR DESCRIPTION
**Pull request by felicianotech**
Completing the items I have from the tracker.

### CODE REVIEW

File: jekyll/_cci2/databases.md [hunk 1]
```diff
@@ -31,27 +31,25 @@ In the primary image the config defines an environment variable with the `enviro
...
 ```
 
-This Postgres image in the example is slightly modified already with `-ram` at the end. It runs in-memory so it does not  hit the disk and that will significantly improve the testing performance on this PostgreSQL database by using this image.
-
 {% raw %}
 
 ```yml
-version: 2
+version: 2.1
 jobs:
```
Comment (kbatuigas) [inline jekyll/_cci2/databases.md:46]: Do we need to include the `orbs` stanza as well where we invoke `circleci/browser-tools`?

**Review by kbatuigas — COMMENTED** [review]
@felicianotech I saw that the postgres and python images are still in `circleci` and not the `cimg` namespace so I want to double check if that's intended.

File: jekyll/_cci2/databases.md (hunk 1 above)
Comment (felicianotech) [reply jekyll/_cci2/databases.md:46]: No, I updated it to remove browsers completely.

**Comment by felicianotech** [comment]
@kbatuigas I don't know what I was smoking when I made this PR but clearly I made a lot of mistakes. I've corrected the things you point out, thank you.

File: jekyll/_cci2/databases.md [hunk 2]
```diff
@@ -101,17 +99,12 @@ When the database service spins up, it automatically creates the database `circl
...
 
-The default `circleci/postgres` Docker image uses regular persistent storage on disk.
-Using `tmpfs` may make tests run faster and may use fewer resources. To use a variant
-leveraging `tmpfs` storage, just append `-ram` to the `circleci/postgres` tag (i.e.,
-`circleci/postgres:9.6-alpine-ram`).
-
-PostGIS is also available and can be combined with the previous example:
-`circleci/postgres:9.6-alpine-postgis-ram`
+PostGIS is available and can be used like this:
+`circleci/postgres:12.0-postgis`
```
Comment (kbatuigas) [inline jekyll/_cci2/databases.md:107]:
```suggestion
`cimg/postgres:12.0-postgis`
```

**Review by kbatuigas — CHANGES_REQUESTED** [review]

**Review by kbatuigas — APPROVED** [review]

**Pull request merged**